### ISSUE TITLE
Give contributors more visibility

### ISSUE THREAD

**Issue by agoscinski** [lab-cosmo/scikit-matter#180]
Similar as in https://github.com/lab-cosmo/equistore/pull/157 we can give contributors more visibility by adding a section to the README and adding images and also introduce a code of conduct
![scikit-matter-contributors](https://contrib.rocks/image?repo=lab-cosmo/scikit-matter)

**Comment by PicoCentauri** [comment]
Yes, very good idea! After we merged #179 my plan was to revise the `README.rst` to avoid duplication. When we do this we can introduce a section for the contributors.
